code/modules/organs/prosthetics/prosthetics_manufacturer_models.dm

```diff
@@ -15,8 +15,13 @@
 	is_robotic = FALSE
 	modular_limb_tier = MODULAR_BODYPART_ANYWHERE
 	bodytype_category = BODYTYPE_HUMANOID
-	material = /decl/material/solid/organic/wood
+	organ_material = /decl/material/solid/organic/wood
 	required_map_tech = MAP_TECH_LEVEL_MEDIEVAL
 	uid = "bodytype_prosthetic_wooden"
 
 DEFINE_ROBOLIMB_MODEL_TRAITS(/decl/bodytype/prosthetic/wooden, pirate, 0, "wooden")
+
+// Dummy/stub prosthetic type for augment implants.
+/decl/bodytype/prosthetic/augment
+	name = "Augmetic"
```

Comment:
i don't think this is a typo but `augmetic` still made me chuckle. apparently it's a 40k thing lol